docs/referenceConf.js

```diff
@@ -49,6 +49,10 @@ exports.config = {
   // connect to an already running instance of Selenium. This usually looks like
   // seleniumAddress: 'http://localhost:4444/wd/hub'
   seleniumAddress: null,
+  // The selenium session id allows Protractor to attach to an existing selenium
+  // browser session. The selenium session is maintained after the test has
+  // completed. Ignored if seleniumSessionId is null.
```

Comment:
Github ate my comment :(

This should be "Ignored if seleniumAddress is null", right?